src/lib.rs

```diff
@@ -41,16 +41,21 @@ pub fn from_hex(v: String) -> Result<Vec<u8>, Error> {
 /// Derives a public DH key from a static DH secret.
 /// sk must be 64 bytes in length or an error will be returned.
 pub fn public_key_from_secret(sk: Vec<u8>) -> Result<Vec<u8>, Error> {
```

Comment:
```suggestion
pub fn public_key_from_secret(secret_key: Vec<u8>) -> Result<Vec<u8>, Error> {
```

If we're renaming things anyways 😄 

We'll also want to update the doc comment